propagators/opentelemetry-propagator-opentracing/test/OpenTracingPropagator.test.ts

```diff
@@ -0,0 +1,304 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  defaultTextMapGetter,
+  defaultTextMapSetter,
+  INVALID_SPANID,
+  INVALID_TRACEID,
+  getSpanContext,
+  setSpanContext,
+  SpanContext,
+  TraceFlags,
+  Baggage,
+  setBaggage,
+  getBaggage,
+} from '@opentelemetry/api';
+import { ROOT_CONTEXT } from '@opentelemetry/context-base';
```

Comment:
I think `ROOT_CONTEXT` is also exported by API so you could get rid of context-base

Reply:
i got rid of the dependency. thanks for pointing that out!